src/promnesia/sources/hypothesis.py

```diff
@@ -1,12 +1,23 @@
-'''
+"""
 Uses HPI [[https://github.com/karlicoss/HPI/blob/master/doc/MODULES.org#myhypothesis][hypothesis]] module
-'''
-from ..common import Visit, Results, logger, Loc
+"""
+from ..common import Loc, Results, Visit, extract_urls
+
+
+def _harvest_text(text, part_name, visit_base) -> Results:
```

Comment:
small nits: 
- I usually try to keep these helpers below 'index' (so you see 'index' the first thing) when you open the module,which is the most important
- if there are multiple arguments of same type (like text/part_name), I usually force them to be keyword (i.e. `def _harvest_text(*, text, part_name, visit_base)`), otherwise it's easy to confuse them and not notice (and even `mypy` won't help!)
but not big issues, just sharing!

Reply:
Ok, done in fba2044 from #224.